### PR TITLE
[TW-815] Flows - add descriptions under headings in Getting basic values

### PR DESCRIPTION
**Pull request by petewitcher**

### CODE REVIEW

File: src/pages/docs/postman-flows/flows-query-language/conditional-data-selection.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ The following examples use the following JSON data returned by an endpoint:
     }
 ```
 
-## Filter for a customer's recurring subscription payments
+## Filter query results for objects with specific key/value pairs
+
+The example below filters for objects in the `payments` array that have the key/value pair `"description": "recurring subscription"`.
```
Comment (jkonrath-postman) [inline src/pages/docs/postman-flows/flows-query-language/conditional-data-selection.md:59]:
```suggestion
The example below filters for objects in the `payments` array that have the key-value pair `"description": "recurring subscription"`.
```

Comment (jkonrath-postman) [reply src/pages/docs/postman-flows/flows-query-language/conditional-data-selection.md:59]: Globally, I think we use "key-value" instead of "key/value"

**Review by jkonrath-postman — APPROVED** [review]
One little thing, but LGTM!

File: src/pages/docs/postman-flows/flows-query-language/conditional-data-selection.md (hunk 1 above)
Comment (petewitcher) [reply src/pages/docs/postman-flows/flows-query-language/conditional-data-selection.md]: I changed all of them.

**Pull request merged**